de-identification-app/src/test/java/com/ibm/whc/deid/app/endpoint/datamasking/DataMaskingControllerErrorPathTest.java

```diff
@@ -88,14 +88,14 @@ public void testMaskEmptyInput() throws Exception {
     log.info(noContent);
     this.mockMvc
         .perform(post(basePath + "/deidentification")
-            .contentType(MediaType.APPLICATION_JSON_UTF8_VALUE).content(noContent))
+            .contentType(MediaType.APPLICATION_JSON_VALUE).content(noContent))
```

Comment:
Here's the justification for the change in case anyone else wonders :
@deprecated Deprecated as of Spring Framework 5.2 in favor of {@link #APPLICATION_JSON_VALUE}
	 * since major browsers like Chrome
	 * <a href="https://bugs.chromium.org/p/chromium/issues/detail?id=438464">
	 * now comply with the specification</a> and interpret correctly UTF-8 special
	 * characters without requiring a {@code charset=UTF-8} parameter.